src/main/python/openresearch/convert_pubmed_dblp_to_anserini_format.py

```diff
@@ -0,0 +1,196 @@
+import argparse
+import gzip
+import json
+import os
+import time
+from collections import defaultdict
+
+
+def clean(text):
+	return text.replace('\n', ' ').replace('\t', ' ')
```

Comment:
It seems there are more than 4 spaces of identation here, right?

Reply:
actually i used tabs... and tab size is set to be 4 spaces, so i think it should be all right

Reply:
But please use only 4 spaces as mixing tabs and spaces breaks the code in Python 3:
https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces

Reply:
so the official resolved this battle, sad :(  
i used all tabs in the file so it wouldn't break, but all right i will follow pep8 in the future